app/controllers/api/v1/trips_controller.rb

```diff
@@ -4,7 +4,11 @@ class Api::V1::TripsController < ApplicationController
    before_action :filter_user_trips, only: [:index]
 
    def index
-      render json: TripSerializer.new(@trips, { params: {show: false, index: true}})
+      if filter_user_trips.empty?
```

Comment:
why do we need to check if empty? we can just return empty array if users dont have trip yet.